### PR TITLE
Nirekr master

### PR DESCRIPTION
**Pull request by nirekr**
- [ ] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [ ] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

**Comment by fogarl1** [comment]
I could be wrong - you might want to check on the 'build' stage in Jenksfile. I think the latest is:
 stage('Build') {
            steps {
                sh "mvn clean install -Dmaven.repo.local=.repo"
            }

**Comment by nirekr** [comment]
@jccheng78   Could you review the Jenkins file? Not sure about the build stage..

File: Jenkinsfile [hunk 1]
```diff
@@ -38,7 +35,13 @@ pipeline {
         }
         stage('Build') {
             steps {
-                sh "mvn clean install -Dmaven.repo.local=.repo"
+                script {
```
Comment (jccheng78) [inline Jenkinsfile]: This script block should be removed.

File: Jenkinsfile [hunk 2]
```diff
@@ -51,11 +54,6 @@ pipeline {
                 doPwScan()
             }
         }
-        stage('Deploy') {
```
Comment (jccheng78) [inline Jenkinsfile]: The Deploy stage should be kept.

File: Jenkinsfile [hunk 3]
```diff
@@ -38,7 +35,13 @@ pipeline {
         }
         stage('Build') {
             steps {
-                sh "mvn clean install -Dmaven.repo.local=.repo"
```
Comment (jccheng78) [inline Jenkinsfile]: This sh step should be kept.

**Comment by nirekr** [comment]
Thank You @jccheng78 . I thought the parameters stage just needs to be in the stable branch and not in the master one.

**Comment by jccheng78** [comment]
@nirekr , for open-sourced repos, the Deploy_Stage parameter and the Deploy stage (doMvnDeploy) should be in all branches.

**Pull request merged**